README.md

```diff
@@ -10,6 +10,11 @@ $ git clone https://github.com/godexsoft/krabby.git --recurse-submodules
 
 You will also need `SQLite3` development package and `Lua5.1` or newer with its development package installed on your system. Krabby will use whatever `Lua` you have available (through `sol3`).
 
+```
```

Comment:
Maybe add a line saying that this is an example for Ubuntu/Debian? because these commands are different on any linux/mac

Reply:
done